src/lib/rest/rest.h

```diff
@@ -38,7 +38,11 @@
 */
 #define MAX_LEN_IP  64
 
-
+/* ****************************************************************************
+*
+* CONSTANTS RESTINIT - 
+*/ 
+#define   RUSH_PORT 0
```

Comment:
Yes, definitely.
RUSH_PORT is not ZERO 'in itself'.
Either NO_PORT or DFAULT_RUSH_PORT, but NO_PORT (or INVALID_PORT) is better.